doc/en/doctest.rst

```diff
@@ -73,7 +79,9 @@ then you can just invoke ``pytest`` without command line options:
 
     ========================= 1 passed in 0.12 seconds =========================
 
-It is possible to use fixtures using the ``getfixture`` helper::
+It is possible to use fixtures using the ``getfixture`` helper:
+
+.. code-block:: text
```

Comment:
`rst` instead of `text`?

Reply:
I actually tried it first, but those files have `doctest`-syntax (valid python) and plain text as comments (valid for doctests). And no actual `rst` markup. 
As as result there is no improvement in how they look on a web-page.